CHANGELOG.md

```diff
@@ -1,4 +1,32 @@
-## 2.9.2 (Unreleased)
+## 2.10.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.2.2122** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented advanced VPC attachment feature in both **aviatrix_aws_tgw** and **aviatrix_aws_tgw_vpc_attachment**
+2. Implemented updating ``customized_routes`` in **aviatrix_aws_tgw_vpc_attachment**
+3. Implemented a check that ``aws_account_number`` needs to be exactly 12 digits in **aviatrix_account**
+4. Implemented multiple missing parameters in **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
```

Comment:
Maybe specify which attributes? or categorize it to maybe "route filtering and advertising options"

For example:
4. Implemented support for spoke VPC route filtering and advertising options in **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**